bundles/pss-baseline-v2022/README.md

```diff
@@ -26,31 +26,26 @@ Controls](https://kubernetes.io/docs/concepts/security/pod-security-standards/#b
 | SELinux               | pss-baseline-v2022-selinux                                                                      |
 | /proc Mount Type      | pss-baseline-v2022-proc-mount-type                                                              |
 | Seccomp               | pss-baseline-v2022-seccomp                                                                      |
-| Sysctls               | (not currently implemented)                                                                     |
+| Sysctls               | pss-baseline-v2022-sysctls                                                                      |
 
-## Usage
-
-### Fetch the package
-`kpt pkg get https://github.com/GoogleCloudPlatform/acm-policy-controller-library.git/bundles/pss-baseline-v2022`
-Details: https://kpt.dev/reference/cli/pkg/get/
+### (Optional) Preview the policy constraints with kubectl:
+```shell
+kubectl kustomize https://github.com/GoogleCloudPlatform/acm-policy-controller-library.git/bundles/pss-baseline-v2022
+```
 
-### View package content
-`kpt pkg tree pss-baseline-v2022`
-Details: https://kpt.dev/reference/cli/pkg/tree/
+### Apply the policy constraints with kubectl:
```

Comment:
Are there instructions to set enforcement action to `deny`? I'm not sure if I see that.

Reply:
Yeah, they can be found on the more information link: https://cloud.google.com/anthos-config-management/docs/how-to/using-pss-baseline#enforce_pod_security_standards_baseline_policies